awx_collection/tests/integration/targets/workflow_job_template/tasks/main.yml

```diff
@@ -233,8 +233,8 @@
     that:
       - "result is changed"
 
-# Node actions do what this schema command used to do
-# schema: [{"success": [{"job_template": "{{ jt1_name }}"}], "job_template": "{{ jt2_name }}"}]
+# Node actions do what this workfow_nodes command used to do
```

Comment:
minor spelling error on this line and next

Reply:
Updated, decided to leave the old text for the one line, and remove the schema line as I believe that is so old that no one remembers what its trying to say.